system/filesystem.py

```diff
@@ -41,6 +41,12 @@
     description:
     - If yes, allows to create new filesystem on devices that already has filesystem.
     required: false
+  growfs:
+    choices: [ "yes", "no" ]
+    default: "no"
+    description:
+    - If yes, if the block device and filessytem size differ, grow the filesystem into the space. This DOES NOT support shrinking. Will only grow if the FS is mounted.
```

Comment:
The FS should be mounted only in the case of XFS filesystem. We could check if fstype=XFS and volume is mounted in this module, and just resize FS in other cases. Also, while XFS cannot be shrinked, we can rename growfs to resizefs and support shrinking via resize2fs. In this case, we should check that FS is not mounted, and run `e2fsck -f /dev/some_disk` before shrinking. Option to shrink FS makes sense, because `lvol` module supports shrinking.